e2e/pageobjects/ide/Editor.ts

```diff
@@ -298,37 +302,33 @@ export class Editor {
         return lineYCoordinate;
     }
 
-    async clickOnTheTextInDefinedPosition(line: number, column: number) {
-        const additionalShiftingToY = 19;
+    async clickOnLineAndChar(line: number, column: number) {
+        
```

Comment:
redundant line, and `additionalShiftingToX` is about replacing by `ADDITIONAL_SHIFTING_TO_X`